src/Engine/Graphics/OpenGL/RenderOpenGL.cpp

```diff
@@ -453,7 +453,7 @@ void RenderOpenGL::DrawProjectile(float srcX, float srcY, float srcworldview, fl
     v29[1].pos.y = dstY - dstymod;
     v29[1].pos.z = dstz;
     v29[1].rhw = dstrhw;
-    v29[1].diffuse = Color(0, 255, 0); // TODO(captainurist): eeeeeeh???
+    v29[1].diffuse = colorTable.Green; // TODO(captainurist): why Green???
```

Comment:
@pskelton can you take a look? Other vertices in the same function are using White.

Reply:
Very likely I had set it to green for debugging. Should be white